partiql-eval/src/main/kotlin/org/partiql/eval/internal/operator/rel/RelScanIndexedPermissive.kt

```diff
@@ -0,0 +1,53 @@
+package org.partiql.eval.internal.operator.rel
+
+import org.partiql.eval.internal.Record
+import org.partiql.eval.internal.operator.Operator
+import org.partiql.value.BagValue
+import org.partiql.value.CollectionValue
+import org.partiql.value.PartiQLValue
+import org.partiql.value.PartiQLValueExperimental
+import org.partiql.value.int64Value
+import org.partiql.value.missingValue
+
+@OptIn(PartiQLValueExperimental::class)
+internal class RelScanIndexedPermissive(
+    private val expr: Operator.Expr
+) : Operator.Relation {
+
+    private lateinit var iterator: Iterator<PartiQLValue>
+    private var index: Long = 0
+    private var isIndexable: Boolean = true
+
+    override fun open() {
+        val r = expr.eval(Record.empty)
```

Comment:
Not related to this PR, but does this work for subquery?

Reply:
That's a good question. I have a subquery PR out, and when I rebase, I'll take a look at that.